test/test_helper.exs

```diff
@@ -1,8 +1,2 @@
-ExUnit.configure(exclude: [:redis_dependent])
 ExUnit.start()
-
-{:ok, files} = File.ls("./test/support")
-
-Enum.each(files, fn file ->
-  Code.require_file("support/#{file}", __DIR__)
-end)
+ExUnit.configure(exclude: [:skip])
```

Comment:
If you're changing this so that we always run the redis_dependent tasks, can you cleanup the CI fiile as well: https://github.com/podium/redis_mutex/blob/master/.github/workflows/ci.yml#L73

Think you can remove the `REDIS_TESTS=true` and the `--include=...` flag. Will let us standardize all the github actions files a bit simpler in the future.

Reply:
Thanks for the suggestion. Done.